scripts/test/run-tests.sh

```diff
@@ -148,10 +155,12 @@ else
 
   if [ "$TEST_ALL" == "1" -o "$TEST_PLATFORM" == "raspberrypi3" ]; then
     wget --progress=dot:giga -N ${RASPBIAN_IMAGE_URL} -P input/image/
-    RASPBIAN_IMAGE="${RASPBIAN_IMAGE_URL##*/}"
+    RASPBIAN_IMAGE_COMPRESSED="${RASPBIAN_IMAGE_URL##*/}"
+    RASPBIAN_IMAGE_UNCOMPRESSED="${RASPBIAN_IMAGE_COMPRESSED%.xz}"
+    unxz --force "input/image/${RASPBIAN_IMAGE_COMPRESSED}"
     convert_and_test "raspberrypi3" \
                      "release-1" \
-                     "input/image/${RASPBIAN_IMAGE}" \
+                     "input/image/${RASPBIAN_IMAGE_UNCOMPRESSED}" \
```

Comment:
I think this one (or one of the others) should be kept, because we are testing xz, distinct from gz.

Reply:
Ok. Fair compromise :)